resources/guis.yml

```diff
@@ -181,4 +181,28 @@ guis:
   - Linux
```

Comment:
the script to order the clients worked nicely :)
couldn't get good google-trends result for git2go and pocket git unfortunately...